library/src/main/res/values/bool.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <bool name="isTablet">false</bool>
```

Comment:
Delete this file, I don't think it's needed any longer. Yes? No?

Reply:
Done.